Gem/Code/Source/NetSoakTestSystemComponent.cpp

```diff
@@ -100,6 +103,7 @@ namespace NetSoakTest
     AZ_CVAR(uint16_t, soak_port, 33450, nullptr, AZ::ConsoleFunctorFlags::DontReplicate, "The port that this soak test will bind to for game traffic");
     AZ_CVAR(ProtocolType, soak_protocol, ProtocolType::Udp, nullptr, AZ::ConsoleFunctorFlags::DontReplicate, "Soak test protocol");
     AZ_CVAR(SoakMode, soak_mode, SoakMode::Loopback, nullptr, AZ::ConsoleFunctorFlags::DontReplicate, "Soak test mode");
+    AZ_CVAR(AZ::TimeMs, soak_runtime, AZ::Time::ZeroTimeMs, nullptr, AZ::ConsoleFunctorFlags::DontReplicate, "How long to run the soak test for before dumping stats");
```

Comment:
```suggestion
    AZ_CVAR(AZ::TimeMs, soak_runtime, AZ::Time::ZeroTimeMs, nullptr, AZ::ConsoleFunctorFlags::DontReplicate, "How long to run (in milliseconds) the soak test for before dumping stats");
```

Reply:
Another thing that might be helpful if folks don't read docs too carefully, I usually append the units to the variable name itself, so soak_runtimeSec, soak_runtimeMs, soak_runtimeUs, soak_runtimeNs, etc..

Reply:
Done